src/xdocs/config_whitespace.xml

```diff
@@ -2268,6 +2268,27 @@ public long toMicros(long d) { return d / (C1 / C0); }
   &lt;property name=&quot;tokens&quot; value=&quot;COMMA, SEMI&quot;/&gt;
 &lt;/module&gt;
         </source>
+              <p>Example:</p>
```

Comment:
Please add example of behavior on default config.

Please share CLI output of config and example code to prove that Check is actually behave like shown in your example

Reply:
<pre><font color="#89E234"><b>➜  </b></font><font color="#34E2E2"><b>check_style_test</b></font> <font color="#729ECF"><b>git:(</b></font><font color="#EF2828"><b>master</b></font><font color="#729ECF"><b>) </b></font><font color="#FBE84F"><b>✗</b></font> cat MyTest.java  
public class MyTest {
    public static void myTest() {
        if (foo) { // OK
            //...
        } else if(bar) { // violation
            //...
        }

        testMethod(foo, bar); // OK
        testMethod(foo,bar);  // violation

        for (;;) {} // OK
    }
}<span style="background-color:#EFEFEF"><font color="#000000"><b>%</b></font></span>                                                                                                                                                                                                                                             <font color="#89E234"><b>➜  </b></font><font color="#34E2E2"><b>check_style_test</b></font> <font color="#729ECF"><b>git:(</b></font><font color="#EF2828"><b>master</b></font><font color="#729ECF"><b>) </b></font><font color="#FBE84F"><b>✗</b></font> checkstyle -c my_checks.xml MyTest.java | grep WhitespaceAfter
[WARN] /home/nick/check_style_test/MyTest.java:5:16: &apos;if&apos; is not followed by whitespace. [<font color="#EF2828"><b>WhitespaceAfter</b></font>]
[WARN] /home/nick/check_style_test/MyTest.java:10:23: &apos;,&apos; is not followed by whitespace. [<font color="#EF2828"><b>WhitespaceAfter</b></font>]
<font color="#89E234"><b>➜  </b></font><font color="#34E2E2"><b>check_style_test</b></font> <font color="#729ECF"><b>git:(</b></font><font color="#EF2828"><b>master</b></font><font color="#729ECF"><b>) </b></font><font color="#FBE84F"><b>✗</b></font> cat my_checks.xml | grep WhitespaceAfter
	&lt;module name=&quot;<font color="#EF2828"><b>WhitespaceAfter</b></font>&quot;/&gt;</pre>

Reply:
<pre><font color="#89E234"><b>➜  </b></font><font color="#34E2E2"><b>check_style_test</b></font> <font color="#729ECF"><b>git:(</b></font><font color="#EF2828"><b>master</b></font><font color="#729ECF"><b>) </b></font><font color="#FBE84F"><b>✗</b></font> cat MyTest.java                                               
public class MyTest {
    public static void myTest() {
        if (foo) { // OK
            //...
        } else if(bar) { // violation
            //...
        }

        testMethod(foo, bar); // OK
        testMethod(foo,bar);  // violation

        for (;;) {} // OK
    }
}
<font color="#89E234"><b>➜  </b></font><font color="#34E2E2"><b>check_style_test</b></font> <font color="#729ECF"><b>git:(</b></font><font color="#EF2828"><b>master</b></font><font color="#729ECF"><b>) </b></font><font color="#FBE84F"><b>✗</b></font> cat my_checks.xml | grep WhitespaceAfter
	&lt;module name=&quot;<font color="#EF2828"><b>WhitespaceAfter</b></font>&quot;/&gt;
<font color="#89E234"><b>➜  </b></font><font color="#34E2E2"><b>check_style_test</b></font> <font color="#729ECF"><b>git:(</b></font><font color="#EF2828"><b>master</b></font><font color="#729ECF"><b>) </b></font><font color="#FBE84F"><b>✗</b></font> checkstyle -c my_checks.xml MyTest.java | grep WhitespaceAfter
[WARN] /home/nick/check_style_test/MyTest.java:5:16: &apos;if&apos; is not followed by whitespace. [<font color="#EF2828"><b>WhitespaceAfter</b></font>]
[WARN] /home/nick/check_style_test/MyTest.java:10:23: &apos;,&apos; is not followed by whitespace. [<font color="#EF2828"><b>WhitespaceAfter</b></font>]</pre>

Reply:
thanks a lot, keep ding this format of prove for each code sample.

Now please make sure that documentation is looks like:
```
<<default config>>
<<code sample>>
some words explanation for config below
<<config>>
<<code sample>>
some words explanation for config below
<<config>>
<<code sample>>
```

you have code sample NOT under default config. Default config is at 2259 line.

Reply:
Ok, I see now.  Everything should be in order.  Thanks for your patience!

Reply:
<pre><font color="#89E234"><b>➜  </b></font><font color="#34E2E2"><b>check_style_test</b></font> <font color="#729ECF"><b>git:(</b></font><font color="#EF2828"><b>master</b></font><font color="#729ECF"><b>) </b></font><font color="#FBE84F"><b>✗</b></font> cat MyTest.java                                             
public class MyTest {
    public static void myTest() {
	int a; int b;		 	// OK
	int a;int b;			// violation
		
        testMethod(foo, bar); 	// OK
        testMethod(foo,bar);  	// violation

        for (;;) {} // OK
    }
}
<font color="#89E234"><b>➜  </b></font><font color="#34E2E2"><b>check_style_test</b></font> <font color="#729ECF"><b>git:(</b></font><font color="#EF2828"><b>master</b></font><font color="#729ECF"><b>) </b></font><font color="#FBE84F"><b>✗</b></font> cat my_checks.xml | sed -n &apos;/WhitespaceAfter/{p;n;p;n;p}&apos;     
	&lt;module name=&quot;WhitespaceAfter&quot;&gt;
  	&lt;property name=&quot;tokens&quot; value=&quot;COMMA, SEMI&quot;/&gt;
	&lt;/module&gt;
<font color="#89E234"><b>➜  </b></font><font color="#34E2E2"><b>check_style_test</b></font> <font color="#729ECF"><b>git:(</b></font><font color="#EF2828"><b>master</b></font><font color="#729ECF"><b>) </b></font><font color="#FBE84F"><b>✗</b></font> checkstyle -c my_checks.xml MyTest.java | grep WhitespaceAfter
[WARN] /home/nick/check_style_test/MyTest.java:4:14: &apos;;&apos; is not followed by whitespace. [<font color="#EF2828"><b>WhitespaceAfter</b></font>]
[WARN] /home/nick/check_style_test/MyTest.java:7:23: &apos;,&apos; is not followed by whitespace. [<font color="#EF2828"><b>WhitespaceAfter</b></font>]</pre>

Reply:
OK. I think I've got everything taken care of here, thanks for being patient!